src/main/java/klerer/rijks/ImageFrame.java

```diff
@@ -0,0 +1,25 @@
+package klerer.rijks;
+
+import javax.imageio.ImageIO;
+import javax.swing.*;
+import java.awt.*;
+import java.net.URL;
+
+public class ImageFrame extends JFrame {
+    public ImageFrame(String title, String artist, String imageUrl) throws Exception {
+        setTitle(title + "by" + artist);
+        setSize(1050, 850);
+        setDefaultCloseOperation(JFrame.DISPOSE_ON_CLOSE);
+        setLayout(new BorderLayout());
+
+        URL url = new URL(imageUrl);
+        Image image = ImageIO.read(url);
+        Image scaledImage = image.getScaledInstance(700, -1, Image.SCALE_DEFAULT);
```

Comment:
If the width of the image is 700, why are you making the size of the window 1050?

Reply:
I changed it but i made it that way so you can see the image